build.subr

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# Shell script for building Phoenix-RTOS ports
+#
+# Common functions
+#
+# Copyright 2024 Phoenix Systems
+# Author: Daniel Sawka
+#
+
+# b_port_download(baseurl, filename, [url_filename])
+b_port_download() {
+	local baseurl="$1"
+	local filename="$2"
+	local url_filename="${3:-$filename}"
+
+	if [ ! -f "${PREFIX_PORT}/${filename}" ]; then
+		if ! wget "${baseurl}${url_filename}" -O "${PREFIX_PORT}/${filename}.part" --no-check-certificate; then
+			wget "${PORTS_MIRROR_BASEURL}${filename}" -O "${PREFIX_PORT}/${filename}.part" --no-check-certificate
+		fi
+
+		if [ -f "${PREFIX_PORT}/checksums.sha256" ]; then
```

Comment:
we download one file but expect checksums to have multiple entries? Are they for supporting multiple versions of the port?

Reply:
We download one file, but may download N files (that's the case in lua). I just put checksums into one file, but maybe it is better to have separate checksum files named `downloaded_file-1.2.3.tar.gz.sha256` in port's root directory?

Reply:
As discussed, opted for separate checksum files stored in _checksums_ directory.